manuscripts.txt

```diff
@@ -1,79 +1,79 @@
-10.1101/2022.06.02.494625
-10.1101/2022.06.17.496451
-10.1101/2022.06.24.497502
-10.1101/2022.06.30.498369
-10.1101/2022.07.13.499930
-10.1101/2022.07.21.500925
-10.1101/2022.07.22.501195
-10.1101/2022.07.26.501569
-10.1101/2022.09.08.507137
-10.1101/2022.09.30.510381
-10.1101/2022.10.17.512253
-10.1101/2022.10.21.513138
-10.1101/2022.10.21.513195
-10.1101/2022.10.25.513688
-10.1101/2022.10.28.514241
-10.1101/2022.10.29.514266
-10.1101/2022.10.30.514301
-10.1101/2022.11.01.514717
-10.1101/2022.11.08.515698
-10.1101/2022.11.08.515699
-10.1101/2022.11.09.515847
-10.1101/2022.11.10.516056
-10.1101/2022.11.11.516105
-10.1101/2022.11.14.516431
-10.1101/2022.11.18.517042
-10.1101/2022.11.23.517579
-10.1101/2022.11.24.517802
-10.1101/2022.11.25.517922
-10.1101/2022.12.01.518662
-10.1101/2022.12.06.519376
-10.1101/2022.12.07.519455
-10.1101/2022.12.15.520660
-10.1101/2022.12.20.521179
-10.1101/2022.12.20.521208
-10.1101/2023.01.03.522550
-10.1101/2023.01.04.522718
-10.1101/2023.01.06.523006
-10.1101/2023.01.11.523553
-10.1101/2023.01.15.524087
-10.1101/2023.01.23.525177
-10.1101/2023.01.26.525679
-10.1101/2023.02.10.528013
-10.1101/2023.02.27.530167
-10.1101/2023.02.08.527616
-10.1101/2023.02.02.526762
-10.1101/2023.02.08.527633
-10.1101/2023.01.26.525761
-10.1101/2023.02.14.528498
-10.1101/2020.01.28.922773
-10.1101/2023.02.01.526734
-10.1101/2023.02.22.529599
-10.1101/2023.02.27.530236
-10.1101/2023.02.20.529267
-10.1101/2023.02.15.528756
-10.1101/2023.02.23.529644
-10.1101/2023.02.23.529713
-10.1101/2023.02.23.529669
-10.1101/2023.02.15.528649
-10.1101/2023.02.21.529382
-10.1101/2021.05.12.443782
-10.1101/2022.12.09.519720
-10.1101/2023.02.27.530196
-10.1101/2023.02.01.526652
-10.1101/2023.03.06.531323
-10.1101/2023.02.22.529521
-10.1101/2023.02.21.529339
-10.1101/2023.01.26.525764
-10.21203/rs.3.rs-2200020/v3
-10.21203/rs.3.rs-2197876/v1
+52299: 10.1101/2022.05.28.493855
+77558: 10.1101/2022.03.04.482974
+80494: 10.1101/2022.06.24.497502
+80729: 10.1101/2022.05.30.22275761
+80984: 10.1101/2022.06.30.498369
+81274: 10.1101/2020.07.27.223354
+81535: 10.1101/2021.11.12.468444
+81727: 10.1101/2022.07.21.500925
+81926: 10.1101/2022.07.26.501569
+84141: 10.1101/2021.09.24.461751
+84338: 10.1101/2022.11.01.514717
+84355: 10.1101/2022.09.30.510381
+84364: 10.1101/2022.10.17.512253
+84385: 10.1101/2022.11.10.516056
+84427: 10.1101/2022.10.21.513195
+84537: 10.1101/2022.11.11.516105
+84553: 10.1101/2022.11.14.516431
+84570: 10.1101/2023.01.03.522550
+84604: 10.1101/2022.07.13.499930
+84628: 10.1101/2022.10.28.514241
+84630: 10.1101/2022.07.22.501195
+84747: 10.1101/2022.11.25.517922
+84792: 10.1101/2022.11.24.517802
+84831: 10.1101/2022.10.30.514301
+84855: 10.1101/2021.06.21.449261
+84874: 10.1101/2022.10.29.514266
+84875: 10.1101/2022.11.08.515699
+84918: 10.1101/2022.12.01.518662
+84921: 10.1101/2022.04.01.486717
+84926: 10.1101/2022.06.17.496451
+85069: 10.1101/2022.12.07.519455
+85111: 10.1101/2022.11.08.515698,10.1101/2022.11.08.515698
```

Comment:
If we have manuscript with multiple versions then repeat the preprint doi. If the preprint doi is different for the versions then just add the new preprint doi comma separated.